src/components/UserLogo.jsx

```diff
@@ -0,0 +1,7 @@
+export default function UserLogo({ user }) {
+  return (
+    <>
+      <h2 className="header-user-logo"> Welcome {user}</h2>
```

Comment:
You could call the useEffect in here instead of passing it down from the parent component.